lib/yarp.rb

```diff
@@ -71,7 +71,7 @@ def copy(**options)
 
     # Returns a string representation of this location.
     def inspect
-      "#<YARP::Location @start_offset=#{@start_offset} @length=#{@length}>"
+      "#<YARP::Location @start_offset=#{@start_offset} @length=#{@length} start_line=#{start_line}>"
```

Comment:
This change I'm slightly unsure of, since `start_line` is not an instance variable. However, this output seems helpful to me for cases like printing error messages.

Reply:
It's only debug code, I think it's fine.